client/src/css/Alternative.css

```diff
@@ -1,43 +1,45 @@
 .Alternative {
-  margin: 1rem 0;
-  background: #6b71a5;
-  color: #ffffff;
-  font-size: 1.8rem;
+  margin: 0.5rem 0;
+  background-color: #6B71A5;
+  color: #FFF;
   display: flex;
-  box-shadow: 0px -2px 0px 0px rgba(0, 0, 0, 0.25) inset,
-              0px 2px 1px 0px rgba(0, 0, 0, 0.2);
+  box-shadow: 0 -0.125rem 0 0 rgba(0, 0, 0, 0.25) inset,
+              0 0.125rem 0.0625rem 0 rgba(0, 0, 0, 0.2);
 }
 
 .Alternative:last-of-type {
   margin-bottom: 0;
 }
 
 .Alternative:hover {
-  background: #8d93c7;
+  background-color: #8D93C7;
 }
 
 .Alternative--selected {
-  background: #4d516f;
+  background-color: #4D516F;
 }
 
 .Alternative--selected:hover {
-  background: #575c7d;
+  background-color: #575C7D;
 }
 
 .Alternative--disabled {
-  font-size: 1.5rem;
   box-shadow: none;
-  background: #f5f5f5;
-  color: #717171;
+  background-color: #F5F5F5;
+  color: #555;
 }
 
 .Alternative--disabled:hover {
-  background: #f5f5f5;
+  background-color: #F5F5F5;
 }
 
 .Alternative label {
   flex-grow: 1;
-  padding: 1rem 3rem;
+  padding: 1rem;
+}
+
+.Alternative:hover label {
+  cursor: pointer;
```

Comment:
`cursor: pointer` should only be used for anchors. 

I made an exception for the header buttons for consistency.